proto/workflow.proto

```diff
@@ -192,6 +188,9 @@ message ExecuteWorkflowResponse {
 
   // A list of the actions executed by the API.
   repeated ActionStatus action_statuses = 3;
+
+  // DEPRECATED: Use the action_statuses field.
+  string invocation_id = 2 [deprecated = true];
```

Comment:
uber nit: can you move this back up above `ActionStatus` just so the field number ordering is still (1, 2, 3)